peripherals/rcc/rcc_l4.yaml

```diff
@@ -13,3 +45,44 @@ RCC:
       Range24M: [0b1001, "range 9 around 24 MHz"]
       Range32M: [0b1010, "range 10 around 32 MHz"]
       Range48M: [0b1011, "range 11 around 48 MHz"]
+
+  CCIPR:
+    ADCSEL:
+      NoClock: [0, "No clock selected"]
+      PLLSAI1: [1, "PLLSAI1 clock selected"]
+      PLLSAI2: [2, "PLLSAI2 clock selected"]
+      SYSCLK: [3, "SYSCLK clock selected"]
+    CLK48SEL:
+      HSI48: [0, "HSI48 clock selected (only for STM32L49x/L4Ax devices, otherwise no clock selected)"]
```

Comment:
The HSI48 option is also valid for RM0394 parts (STM32L41x/L42x/L43x/L44x/L45x/L46x)